omod/src/main/webapp/fragments/summary/registrationSummary.gsp

```diff
@@ -47,7 +47,7 @@
                     <h3>${ ui.message("coreapps.clinicianfacing.overallActions") }</h3>
                     <%
                         overallActions.each { ext -> %>
-                            <a href="${ ui.escapeJs(ext.url("/" + ui.contextPath(), appContextModel, returnUrl)) }" id="${ ext.id }">
+                            <a href="${ ui.encodeJavaScript(ext.url("/" + ui.contextPath(), appContextModel, returnUrl)) }" id="${ ext.id }">
```

Comment:
Do you consider using `ui.encodeHtmlAttribute(String)` instead for html attributes? see => https://github.com/OWASP/owasp-java-encoder/blob/master/core/src/main/java/org/owasp/encoder/Encode.java#L299-L301